app/templates/index.html.orig

```diff
@@ -0,0 +1,173 @@
+<!--
```

Comment:
Did you mean to add this .orig file? Or is this like a swap file and it got added because you had it open at commit time?

Reply:
that file actually just appeared when I rebased and I forgot to deal with it